src/mcp_server_helm/core/server.py

```
@@ -0,0 +1,281 @@
+    basic, dependencies, get, release, repository,
+    registry, search, show, package, plugin
+)
+
+logger = logging.getLogger(__name__)
+
+
+class HelmMCPServer:
+    """
+    MCP Server for Helm commands.
+    """
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)